itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenariosAcrossInstances.java

```diff
@@ -944,96 +944,6 @@ public void testIncrementalDumpMultiIteration() throws Throwable {
     Assert.assertEquals(IncrementalLoadTasksBuilder.getNumIteration(), numEvents);
   }
 
-  @Test
-  public void testIfCkptAndSourceOfReplPropsIgnoredByReplDump() throws Throwable {
```

Comment:
This was testing that source of replication properties are ignored by replication while the custom ones are not. Can we retain that part?

Reply:
Agree, added